src/gpgpu-sim/gpu-cache.cc

```diff
@@ -280,7 +280,7 @@ enum cache_request_status tag_array::probe(new_addr_type addr, unsigned &idx,
     }
     if (!line->is_reserved_line()) {
       if (!line->is_modified_line() ||
-          100 * m_dirty / (m_config.m_nset * m_config.m_assoc) >=
+          m_dirty / (m_config.m_nset * m_config.m_assoc * 100) >=
```

Comment:
please write it as I said:        (m_dirty / (m_config.m_nset * m_config.m_assoc )) * 100 >=